### ISSUE TITLE
Pixeled

### ISSUE THREAD

**Issue by Lalamcr** [microsoft/vscode#124226]
Issue Type: <b>Bug</b>

I got the left side bar and bottom bar pixeled 

VS Code version: Code 1.56.2 (054a9295330880ed74ceaedda236253b4f39a335, 2021-05-12T17:13:13.157Z)
OS version: Windows_NT x64 10.0.19042

<details>
<summary>System Info</summary>

|Item|Value|
![VScodeIssue](https://user-images.githubusercontent.com/84496059/118933429-6c215c00-b906-11eb-931e-e0173a8453d6.jpg)
![VScodeIssue](https://user-images.githubusercontent.com/84496059/118933492-7fccc280-b906-11eb-94b7-e66eccc15ae8.jpg)

|---|---|
|CPUs|Intel(R) Core(TM) i7-7700HQ CPU @ 2.80GHz (8 x 2808)|
|GPU Status|2d_canvas: enabled<br>gpu_compositing: enabled<br>multiple_raster_threads: enabled_on<br>oop_rasterization: enabled<br>opengl: enabled_on<br>rasterization: enabled<br>skia_renderer: enabled_on<br>video_decode: enabled<br>vulkan: disabled_off<br>webgl: enabled<br>webgl2: enabled|
|Load (avg)|undefined|
|Memory (System)|19.84GB (11.23GB free)|
|Process Argv||
|Screen Reader|no|
|VM|0%|
</details>Extensions: none<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vsreu685:30147344
python383cf:30185419
vspyt653:30270858
pythonvspyt602:30300191
vspor879:30202332
vspor708:30202333
vspor363:30204092
pythonvspyt639:30300192
pythontb:30283811
vspre833cf:30267465
pythonptprofiler:30281270
vsdfh931:30280409
vshan820:30294714
pythondataviewer:30285071
vscus158cf:30286554
vscgsv2:30301612
vscorehov:30302759
bridgeflightcf:30302070
vscod805:30301674

```

</details>

**Comment by roblourens** [comment]
Does it help to start vscode with `--disable-gpu`?

**Comment by Lalamcr** [comment]
> Does it help to start vscode with `--disable-gpu`?

Help a little, at least it decreased the number of times that happens.

**Comment by deepak1556** [comment]
Couple of things to check:

1) Do you see the issue when opening https://vscode-web-test-playground.azurewebsites.net/ on your chrome or edge browser ?
2) Do you see the issue when you have the setting either `"window.titleBarStyle": "native"` or `"window.titleBarStyle": "custom"`
3) Are you running vscode as administrator ?

**Comment by Systemcluster** [comment]
@deepak1556 I don't think this is the same issue as reported in #148261. The issue described in #148261 exclusively affects overflowing scroll views (editor, tabs bar, etc.), where lines appear duplicated or the scrollview is rendered twice. 
![image](https://user-images.githubusercontent.com/2847328/206953485-8a5fe0da-d5f0-47e2-88c5-45be0358b1ff.png)
That issue is simple to temporarily resolve by just causing a rerender, e.g. by tabbing out or selecting text.
Additional information about the scroll view render issue:
- Not reliable to reproduce, but most often occurs in the editor when scrolling a long file all the way to the top, or after scrolling in the tab bar when having many tabs open.
- Appears with both `titleBarStyle` custom and native.
- Doesn't seem to appear with gpu acceleration disabled.

**Comment by starball5** [comment]
Related on Stack Overflow (which I use as a duplicate-target): [Why does VS Code give a chip-like / distorted display when I hover over it with my mouse?](https://stackoverflow.com/q/72962855/11107541)
